package/MDAnalysis/analysis/align.py

```diff
@@ -541,6 +544,120 @@ def alignto(mobile, reference, select=None, weights=None,
     return old_rmsd, new_rmsd
 
 
+@due.dcite(
+        Doi("10.1021/acs.jpcb.7b11988"),
+        description="Iterative Calculation of Opimal Reference",
+        path="MDAnalysis.analysis.align.iterative_average"
+)
+def iterative_average(
+    mobile, reference=None, select='all', weights=None, niter=100,
+    eps=1e-6, verbose=False, **kwargs
+):
+    """Iteratively calculate an optimal reference that is also the average
+    structure after an RMSD alignment.
+
+    The optimal reference is defined as average
+    structure of a trajectory, with the optimal reference used as input.
+    This function computes the optimal reference by using a starting
+    reference for the average structure, which is used as the reference
+    to calculate the average structure again. This is repeated until the
+    reference structure has converged. :footcite:p:`Linke2018`
```

Comment:
I just checked https://mdanalysis--4524.org.readthedocs.build/en/4524/documentation_pages/analysis/align.html#MDAnalysis.analysis.align.iterative_average and the reference  `[1]` does not resolve.  Please check how to use footnote citations, e.g., as used in https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/analysis/bat.py 

I think you need to remove the reference from references and instead add a 
```reST

References
----------

.. footbibliography::

```
at the end of the module docs.